docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -12,23 +12,28 @@ The search is case-sensitive by default in all these functions. There are separa
 
 ## position(haystack, needle), locate(haystack, needle) {#position}
 
-Returns the position (in bytes) of the found substring in the string, starting from 1.
+Searches for the substring `needle` in the string `haystack`.
+
+Returns the position (in bytes) of the found substring in the string, starting from 1. 
 
 For a case-insensitive search, use the function [positionCaseInsensitive](#positioncaseinsensitive).
 
 **Syntax**
 
 ``` sql
-position(haystack, needle[, start_pos])
-```
+position((haystack, needle[, start_pos]) | (needle IN haystack))
```

Comment:
It looks a bit weird and hard to read for users. Maybe it would be better to write two variants of syntax separately?

Reply:
> It looks a bit weird and hard to read for users. Maybe it would be better to write two variants of syntax separately?

Interesting idea. Also for dividing examples below.